app/src/main/java/com/hover/stax/transfers/TransferFragment.java

```diff
@@ -206,11 +212,22 @@ protected void onContactSelected(int requestCode, StaxContact contact) {
 		}
 	};
 
-	private void load(Request r) {
+	private void loadAndIndicateFieldState(Request r) {
 		channelDropdownViewModel.setChannelFromRequest(r);
 		amountInput.setText(r.amount);
 		recipientAutocomplete.setText(r.requester_number);
 		transferViewModel.setEditing(r.amount == null || r.amount.isEmpty());
+		indicateFieldState(r.amount, r.requester_number);
 		Amplitude.getInstance().logEvent(getString(R.string.loaded_request_link));
 	}
+	private void indicateFieldState(String amount, String requesterNum) {
+		if(amount == null || amount.isEmpty()) amountEntry.requestFocus();
+		if(requesterNum !=null && !requesterNum.isEmpty()) {
+			new Handler().postDelayed(() -> {
```

Comment:
Definitely don't this. Why is there a delay? Why does it need a thread? Why does this only happen if the requester number exists? Why are we setting the state manually here, but we aren't setting the success state for channel dropdown in any other case? Why does the setSuccess method take an empty String at all, particularly since it is never used any other way?

Reply:
Handler shouldn't be here, I used this while trying to figure out why channelDrop state kept changing while loading. And got the comment on field state not existing as its own object. Fix coming in next commit